### ISSUE TITLE
[VCWG] CR update for vc-data-model (shortname was verifiable-claims-data-model)

### ISSUE THREAD

**Issue by burnburn** [w3c/transitions#152]
# Link to updated CR draft
https://w3c.github.io/vc-data-model/CR/2019-07-25/ 
For informational purposes, an auto-generated diff from the previous CR is at https://w3c.github.io/vc-data-model/CR/2019-07-25/diff.html

# Link to GitHub files for updated CR draft
https://github.com/w3c/vc-data-model/tree/gh-pages/CR/2019-07-25 

# Link to group's decision to request transition
https://www.w3.org/2019/07/23-vcwg-minutes.html#resolution01 
https://www.w3.org/2019/07/23-vcwg-minutes.html#resolution03 (addresses second conditional clause in above vote)

# Link to previous Candidate Recommendation transition request
https://lists.w3.org/Archives/Member/chairs/2019JanMar/0143.html 

# Substantive changes

There was a substantive change to the JWT section regarding the use of iat and nbf. A bug fix was made to the specification to use nbf instead of iat.

There was a substantive change to the JWT section regarding the use of iss. A bug fix was made to the specification to ensure that iss is set to the holder when a JWT-based verifiable presentation is processed.

# Any changes in normative references?
The JSON-LD reference was changed from version 1.0 to version 1.1.  However, the features of JSON-LD used in this specification are unchanged in JSON-LD 1.1, so no implementation is required to change as a result.

Additionally, the reference to RFC 7516 was made informative.

Finally, the specification already stated its use of RFC 2119 for normative language (as part of the standard boilerplate built into ReSpec).  An official reference has been added to RFC 8174 as an update to that RFC.

# Any changes in requirements?
No.

# Wide Review of substantive changes
Not needed.  The changes are widely considered to be bug fixes with no additional security, privacy, internationalization, etc. consequences.

# Issues status
In summary, only 3 issues remain open (526, 704, 705).  See https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22defer%22 
Issues 704 and 705 will be worked on over the next 28 days and are expected to result in non-normative text changes.
Issue 526 is waiting on the SysTeam to update our TR space to use the new TR shortname of vc-data-model, promised to happen at our next publication.

Additionally, there were 3 actual (non-formal) objections.  See below under "Disposition".

Issues received during the first CR review period: https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22CR-phase%22+-label%3A%22Post+CR+comment+period%22+ 
Issues received after the first CR review period (not required to address): https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Post+CR+comment+period%22+ 

Disposition (across all of the above):
- Explicit (non-formal) objections: https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3AObjection+ 
- Closed 7 days after notice of intent to close unless objections: https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Pending+7+Day+Close%22+ 
- Remainder (closed/deferred after submitter approval or other clear indication of wrap-up): https://github.com/w3c/vc-data-model/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22CR-phase%22+-label%3A%22Pending+7+Day+Close%22+-label%3A%22Objection%22+ 

# Formal Objections
None.

# Any changes in implementation information?
The Working Group has obtained reports from nine (9) implementations, with at least two implementations of each feature (other than the substantive changes just made).

# Deadline for further comments
August 21, 2019

# Any changes in patent disclosures?
No.

**Comment by msporny** [comment]
Please make sure that the new shortname is used when publishing this CR. To be clear, the new shortname should be: vc-data-model

The TR link should be:

https://www.w3.org/TR/vc-data-model/

The date-stamped version should be:

https://www.w3.org/TR/2019/CR-vc-data-model-20190725/

The following URL should then be updated to redirect appropriately:

https://www.w3.org/TR/verifiable-claims-data-model/ -> https://www.w3.org/TR/vc-data-model/

**Comment by swickr** [comment]
In Section 4.2 there is some new normative text for the /id/ property.   This might be additional constraints on the property value or might just be elaboration on normative requirements of the previous draft.  Whichever this is might warrant an additional bullet in the SoTD changes list; I leave it to you to decide.

Some Features at Risk appear to no longer be At Risk.  I recommend you include in the SoTD changes list any features that were and are no longer At Risk.

**Comment by swickr** [comment]
Transition approved.
(CR update)

**Comment by burnburn** [comment]
@swickr Oops, somehow all of us missed your comment above.  We weren't ignoring you :)
Regarding your first point, this was a clarification of the existing text in the previous CR.  We don't believe it warrants a separate bullet.  Regarding the Features at Risk, you are right that we should have listed the features no longer at risk, although we did clarify that we have sufficient implementations of all features except those dependent upon the substantive changes made in this document.

Would you still like us to update the SoTD?  Just let us know how you would like us to handle this.

**Comment by swickr** [comment]
Thanks for confirming the /id/ changes are simply clarifications.

As to the no-longer-At-Risk items, though that would have been good/easy to do prior to publication I won't insist that you put the Webmaster through that additional effort now.

**Comment by burnburn** [comment]
Thanks @swickr .

**Comment by ashimura** [comment]
The spec draft has been published as https://www.w3.org/TR/2019/CR-vc-data-model-20190725/

Thanks a lot for your cooperation, all!
